scaffold/cmd/templates/source/go.mod.tpl

```diff
@@ -0,0 +1,8 @@
+module github.com/{{.Org}}/cq-source-{{.Name}}
```

Comment:
Potential future improvement: Maybe we should run `go mod tidy` on the directory after generating so that the `go.sum` gets generated as well. Otherwise you get a slightly confusing error the first time you try to run it

Reply:
we can, or we can bake `go.mod` in. The issue with running `go mod` that it might not be available and it's like a different step.